app/helpers/document_helper.rb

```diff
@@ -12,4 +12,8 @@ def filter_current(documents)
   def archive_reason_collection_for_radio_buttons
     Document.archive_reasons.keys.map { |k| [k, I18n.t("archive_reasons.#{k}")] }
   end
+
+  def override_numbers(document)
```

Comment:
I don't quite understand what this method is doing or for? Maybe it just needs a clearer name?

Reply:
By changing the numbers column type to a string, we are able to get rid of various helper methods (not just this one, but also the methods that split the string etc)